map_frame: map
robot_frame: base_link
max_range: 10.0
resolution: 0.07
prob_hit: 0.99
prob_miss: 0.10
thres_min: 0.49
thres_max: 0.51
publish_pointcloud: true
publish_vis_marker: true
publish_updates: false
publish_overwrites: false
publish_sections: false
apply_raw_sensor_data: false
map_save_dir: ""

remote_sources: [vdb_mapping]
vdb_mapping:
  namespace: vdb_mapping
  apply_remote_updates: false
  apply_remote_overwrites: false
  apply_remote_sections: true

remote_data_sources: [vdb_ground_types]
vdb_ground_types:
  namespace: vdb_mapping
  topic_name: ground_types_cloud
  identifier: data_points
